### PR TITLE
zh-translation: docs/examples/virtual-machines/index.md

### PR DESCRIPTION
**Pull request by MXuDong**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/examples/virtual-machines/index.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+- 部署 [Bookinfo](/zh/docs/examples/bookinfo/) 示例应用（在 `bookinfo` 命名空间下）。
+
+- 按照[虚拟机配置](/zh/docs/setup/install/virtual-machine/#configure-the-virtual-machine)创建一个虚拟机并添加到 `vm` 命名空间下。
+
+## 在虚拟机上运行 MySQL{#running-MySQL-on-the-VM}
+
+您将在虚拟机上安装 MySQL，并将其配置为 ratings 服务的后端。
+
+下列的所有命令都会在虚机上执行。
```
Comment (kebe7jun) [inline content/zh/docs/examples/virtual-machines/index.md:39]:
```suggestion
下列的所有命令都在虚机上执行。
```

File: content/zh/docs/examples/virtual-machines/index.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+在上面的示例中，我们将虚拟机视为一个服务。 您还可以在您的虚拟机中无缝调用 Kubernetes 的服务：
+
+{{< text bash >}}
+$ curl productpage.bookinfo:9080
+...
+<title>Simple Bookstore App</title>
+...
+{{< /text >}}
+
+Istio 的 [DNS 代理](/zh/docs/ops/configuration/traffic-management/dns-proxy/)自动为您的虚机配置 DNS , 允许我们调用 Kubernetes 的主机名。
```
Comment (kebe7jun) [inline content/zh/docs/examples/virtual-machines/index.md:144]:
```suggestion
Istio 的 [DNS 代理](/zh/docs/ops/configuration/traffic-management/dns-proxy/)自动为您的虚机配置 DNS , 允许我们 Kubernetes 的主机名进行访问。
```

File: content/zh/docs/examples/virtual-machines/index.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+创建强制 Bookinfo 使用 ratings 后端的路由规则：
+
+{{< text bash >}}
+$ kubectl apply -n bookinfo -f @samples/bookinfo/networking/virtual-service-ratings-mysql-vm.yaml@
+{{< /text >}}
+
+您可以验证 Bookinfo 应用程序的输出显示的是 Reviewer1 的 1 个星，还是 Reviewer2 的 4 个星，或者更改虚拟机的 ratings 服务并查看结果。
+
+## Reaching Kubernetes services from the virtual machine
```
Comment (kebe7jun) [inline content/zh/docs/examples/virtual-machines/index.md:133]: 没翻译哦。

**Review by kebe7jun — APPROVED** [review]
lgtm

**Pull request merged**